### DISCUSSION TITLE
How to make the popper content shift to right a bit?

### DISCUSSION THREAD

**Discussion by mockingjet** [floating-ui/floating-ui#1947] [Q&A] [answered]
How to make the popper content shift to right a bit?

![image](https://user-images.githubusercontent.com/32839503/200222827-6922f256-e54c-4ad6-a9fc-7c0aa9fbdcb8.png)

```ts
const options = {
    placement: 'bottom-end',
    middleware: [offset(8), flip(), arrow({ element: arrowEl })],
}
```

I add `shift({padding: 12})` but it didn't work.

**Comment by atomiks** [comment, marked as answer]
`offset({mainAxis: 8, crossAxis: 8})`

**Reply by mockingjet** [reply]
I seem to misunderstand offset and shift, thank you.
